pkgs/by-name/gu/guile-goblins/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchurl
+, guile
+, guile-fibers
+, guile-gcrypt
+, texinfo
+, pkg-config
+}:
+stdenv.mkDerivation rec {
+  pname = "guile-goblins";
+  version = "0.11.0";
+
+  src = fetchurl {
+    url =
+      "https://spritely.institute/files/releases/guile-goblins/guile-goblins-${version}.tar.gz";
+    hash = "sha256-1FD35xvayqC04oPdgts08DJl6PVnhc9K/Dr+NYtxhMU=";
+  };
+
+  nativeBuildInputs = [ pkg-config texinfo ];
```

Comment:
Another thing I'd like to check is whether `guile` is needed in `nativeBuildInputs`, by setting `strictDeps = true`. This would make cross-builds easier.

Reply:
So if the build fails with `strictDeps = true`, put `guile` into `nativeBuildInputs` - should I keep `strictDeps = true` after that?

Reply:
Yes, please.